rush.json

```diff
@@ -1,37 +1,255 @@
+/**
+ * This is the main configuration file for Rush.
+ * For full documentation, please see https://rushjs.io
+ */
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/rush.schema.json",
 
+  /**
+   * (Required) This specifies the version of the Rush engine to be used in this repo.
+   * Rush's "version selector" feature ensures that the globally installed tool will
```

Comment:
>selector [](start = 21, length = 8)

multiplexer? What was the official name used?

Reply:
It's "version selector":

[https://github.com/Microsoft/web-build-tools/blob/master/apps/rush/src/RushVersionSelector.ts](https://github.com/Microsoft/web-build-tools/blob/master/apps/rush/src/RushVersionSelector.ts)

---
In reply to: [217226795](https://github.com/Microsoft/rush-example/pull/2#discussion_r217226795) [](ancestors = 217226795)